src/Shell/GettextShell.php

```diff
@@ -349,4 +348,33 @@ private function parseDir($dir) : void
             }
         }
     }
+
+    /**
+     * Extract translations from javascript files using ttag, if available.
+     *
+     * @return void
+     */
+    private function ttagExtract() : void
+    {
+        // check ttag and js folder exist
+        $ttag = 'node_modules/ttag-cli/bin/ttag';
+        $appDir = 'src/Template/Layout/js/app';
```

Comment:
Let's use `src/Template` as JS root dir for now, we still have to decide the best folder layout
[in some projects we are working on `src/Template/Layout/js/app` does not exist...]